frontend/src/components/Pages/YTC/Executor/index.tsx

```diff
@@ -0,0 +1,184 @@
+                    <Flex
+                        id="outputs"
+                        flexDir='column'
+                        alignItems='center'
+                        gridGap={1}
+                        p={2}
+                    >
+                        <FormLabel>
+                            Tokens Received
+                        </FormLabel>
+                        <TokenResult
+                            tokenType="BaseToken"
+                            trancheAddress={userData.trancheAddress}
+                            token={{
+                                name: baseToken.name,
+                                amount: baseTokenAmount
+                            }}
+                        />
+                        <TokenResult
+                            tokenType="YToken"
+                            trancheAddress={userData.trancheAddress}
+                            token={{
+                                name: yieldToken.name,
+                                amount: yieldTokenAmount
+                            }}
+                            baseTokenName={baseToken.name}
+                        />
+                    </Flex>
+                    <Divider
+                        borderColor="background.100"
+                        orientation="horizontal"
+                        borderWidth="thin"
+                    />
+                <ExecutionDetails 
+                    slippageTolerance={slippageTolerance}
+                    estimatedGas={estimatedGas}
+                    percentageReturn={estimatedApy}
+                    minimumReturn={minimumReturn}
+                />
+            </Card>
+            <Button
+                id="approve-calculate-button"
+                rounded="full"
+                bgColor="main.primary"
+                _hover={{
+                    bgColor:"main.primary_hover"
+                }}
+                mt="4"
+                p="2"
+                textColor="text.secondary"
+                onClick={handleExecuteTransaction}
+            >
+                {isLoading ? <Spinner/> : "APE"}
+            </Button>
+        </Flex>
+    )
+}
+
+
+interface ExecutionDetailsProps {
+    slippageTolerance: number,
+    minimumReturn: number,
+    estimatedGas: number,
+    percentageReturn : number | undefined,
+}
+
+const ExecutionDetails: React.FC<ExecutionDetailsProps> = (props) => {
+    const {slippageTolerance, minimumReturn, estimatedGas, percentageReturn} = props;
+
+    return <Flex
+        p={2}
+        px={5}
+        flexDir="column"
+        width="full"
+    >
+        <DetailItem
+            name="Slippage Tolerance:"
+            value={`${slippageTolerance}%`}
+        />
+        <DetailItem
+            name="Minimum YT Received:"
+            value={`${shortenNumber(minimumReturn)}`}
+        />
+        <DetailItem
+            name="Estimated Gas Cost:"
+            value={`${shortenNumber(estimatedGas)} ETH`}
```

Comment:
do we really need these template literals here?
Can it not br smth like value={shortenNumber(estimatedGas)} ETH